components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/APIPersistence.java

```diff
@@ -179,7 +189,21 @@ PublisherAPISearchResult searchAPIsForPublisher(Organization org, String searchQ
      */
     DevPortalAPISearchResult searchAPIsForDevPortal(Organization org, String searchQuery, int start, int offset,
                                     UserContext ctx) throws APIPersistenceException;
-    
+
+    /**
+     * Search APIs to be displayed on Dev Portal API listing
+     *
+     * @param org         Organization the APIs are owned by
+     * @param searchQuery search query
+     * @param start       starting index
+     * @param offset      search offset
+     * @param isAllowDisplayMultipleVersions      whether to display multiple versions
+     * @return Dev Portal API Search Result
+     * @throws APIPersistenceException
+     */
+    DevPortalAPISearchResult searchAPIsForDevPortal(Organization org, String searchQuery, int start, int offset,
```

Comment:
Changing the interface means you will have to update the choreo implementation as well (at least the method signature.) . Since this **isAllowDisplayMultipleVersions** is a configuration value, shall we pass it as a property or config to the persistence module as done in https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java#L254 and access it from the persistenceImpl layer when it is needed